rust/datafusion/src/datasource/csv.rs

```diff
@@ -66,7 +66,10 @@ impl CsvFile {
                 let mut filenames: Vec<String> = vec![];
                 common::build_file_list(path, &mut filenames, options.file_extension)?;
                 if filenames.is_empty() {
-                    return Err(DataFusionError::Plan("No files found".to_string()));
+                    return Err(DataFusionError::Plan(format!(
+                        "No files found at {path}",
```

Comment:
Can we show the file extension it was looking for as well?

Reply:
fixed